### PR TITLE
Ronaru/issue 799

### PR DESCRIPTION
**Pull request by rohitnarula7176**
*Remove dependency on node version 0.10 and 0.11.
#799

### CODE REVIEW

File: lib/adal.js [hunk 1]
```diff
@@ -924,7 +924,7 @@ var AuthenticationContext = (function () {
      * @ignore
      */
     AuthenticationContext.prototype._addHintParameters = function (urlNavigate) {
-        //If you don�t use prompt=none, then if the session does not exist, there will be a failure.
+        //If you don?t use prompt=none, then if the session does not exist, there will be a failure.
```
Comment (manoj-rath) [inline lib/adal.js]: you meant apostrophe?

File: .travis.yml [hunk 2]
```diff
@@ -1,7 +1,3 @@
 language: node_js
 node_js:
-  - "0.11"
-  - "0.10"
-  - "4.7.3"
-  - "5.12.0"
-  - "6.1.0"
+  - "7"
```
Comment (manoj-rath) [inline .travis.yml]: why 7?, why not latest?

Comment (rohitnarula7176) [reply .travis.yml]: Changed it to test the latest.

File: lib/adal.js (hunk 1 above)
Comment (rohitnarula7176) [reply lib/adal.js]: Typo. Done

**Review by manoj-rath — APPROVED** [review]
looks good

**Pull request merged**